### PR TITLE
Remake tools that dont require entire mcdata structure

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Comment by rom1504** [comment]
Can you adapt the readme ?

**Comment by u9g** [comment]
> Can you adapt the readme ?

Done

File: README.md [hunk 1]
```diff
@@ -40,6 +40,15 @@ Make sure to first extract the data folder from Minecraft jar if not done alread
 Next you can extract the actual loot table data to Minecraft-Data using:
 `node extract_lootTables.js <version1,version2,...> <extractedDataFolder> <mcDataFolder>`
```
Comment (rom1504) [inline README.md:41]: This needs to be adapted

Comment (u9g) [reply README.md:41]: why ? it wasn't changed.

**Review by rom1504 — APPROVED** [review]

**Pull request merged**